editor/editor_node.cpp

```diff
@@ -1386,6 +1386,13 @@ int EditorNode::_save_external_resources() {
 }
 
 void EditorNode::_save_scene(String p_file, int idx) {
+	// Imported 3D scenes can't be saved by the Godot editor.
+	// Exit early to avoid displaying a "Requested file format unknown:" alert dialog
+	// when running the project, as running the project saves all opened saves by default.
+	if (p_file.ends_with(".gltf") || p_file.ends_with(".glb") || p_file.ends_with(".dae") || p_file.ends_with(".obj")) {
```

Comment:
I don't like hardcoding those.

I think you can use `EditorSceneImporter::get_extensions()` to get a list of supported extensions for registered scene importer plugins. This would also cover `.fbx` which was missed here.
It would also include `.escn` - not sure if this one needs to be restricted the same way as others? If not, a boolean property should likely be added to `EditorSceneImporter` to define if it's a save-able format.

Reply:
I just checked and both Godot 4.0.alpha and Godot 3.4.4 can't save an ESCN scene. It should be OK to prevent Godot from attempting to save ESCN scenes:

![image](https://user-images.githubusercontent.com/180032/180892231-0058412d-fd31-4954-bd30-364ae4484da4.png)

Reply:
I redone this PR to avoid hardcoding the list of extensions.

For reference, in 4.0.alpha, the list of imported scene extensions is:

```
dae
obj
escn
gltf
glb
blend
```

(+ `fbx` if FBX support is enabled in the editor settings)